static/docs/tutorial/define-ml-pipeline.md

```diff
@@ -118,6 +118,9 @@ and Windows. Some implementation details (like inodes) might differ, but the
 overall DVC behavior is the same. See also [DVC Files and
 Directories](/doc/user-guide/dvc-files-and-directories)
 
```

Comment:
this note above should be update to mention reflinks on modern file systems. I'm not sure why do we need the reference to the DVC files and directories here.

Reply:
Can I remove the reference to DVC files and directories?

Reply:
@kurianbenoy Yes! I think it made it here by mistake. This link is not relevant. There should be a link to some doc that explains everything about link types, something like that one I shared below.